site/docs/guide/getting-started.md

```diff
@@ -150,6 +150,90 @@ in your terminal before you execute `node bot.js`.
 This makes it easier to debug your bot.
 :::
 
+## Getting Started on Bun
+
+> This guide assumes that you have [Bun](https://bun.sh) installed.
+> If you don't know what these things are, check out our [Introduction](./introduction)!
+
+Create a new project and install the `grammy` package.
+Do this by opening a terminal and typing:
+
+```sh
+# Create a new directory and change into it.
+mkdir my-bot
+cd my-bot
+
+# Run bun init to scaffold a new project.
+bun init --yes
+
+# Install grammY.
+bun add grammy
+```
+
+Your folder structure should looks like this:
+
+```asciiart:no-line-numbers
+.
+├── index.ts
+├── .gitignore
+├── node_modules/
+├── package.json
+├── bun.lockb
+├── README.md
+└── tsconfig.json
+```
+
+Now, it's time to open Telegram to create a bot account, and obtain a bot token for it.
+Talk to [@BotFather](https://t.me/BotFather) to do this.
+The bot token looks like `123456:aBcDeF_gHiJkLmNoP-q`.
+It is used to authenticate your bot.
+
+Got the token? You can now code your bot in the `index.ts` file.
+You can copy the following example bot into that file, and pass your token to the `Bot` constructor:
+
+```ts [TypeScript]
+import { Bot } from "grammy";
+
+// Create an instance of the `Bot` class and pass your bot token to it.
+const bot = new Bot(""); // <-- put your bot token between the ""
+
+// You can now register listeners on your bot object `bot`.
+// grammY will call the listeners when users send messages to your bot.
+
+// Handle the /start command.
+bot.command("start", (ctx) => ctx.reply("Welcome! Up and running."));
+// Handle other messages.
+bot.on("message", (ctx) => ctx.reply("Got another message!"));
+
+// Now that you specified how to handle messages, you can start your bot.
+// This will connect to the Telegram servers and wait for messages.
+
+// Start the bot.
+bot.start();
+```
+
+You can now run the bot by executing
+
+```sh
+bun run bot.ts
```

Comment:
Didn't changed that
```
bun run index.ts
```

Reply:
Why not?

Reply:
Actually, it would be much better to create a `bot.ts` file instead of `index.ts`. It makes no sense to do this inconsistently for Bun.

Reply:
> Why not?

Just forgot about it. These are comments addressed to me in the morning 😶‍🌫️

Reply:
> Actually, it would be much better to create a `bot.ts` file instead of `index.ts`. It makes no sense to do this inconsistently for Bun.

Hm, you're right. If all the examples use `bot.ts`, it would be convenient to leave this for association

Reply:
Do we have to use the init script then? Maybe we can just create the files manually or use `npm init`?

Reply:
Yeah, why not. If the Bun has its own initialization that looks more convenient than npm, why we shouldn't use it

Reply:
Because it clearly isn't more convenient, it creates the wrong files

Reply:
What you mean by `creates the wrong files`

Reply:
The files it creates have the wrong content.
- `index.ts` should actually be named `bot.ts`
- `bot.ts` should actually different content
- `tsconfig.json` is wrong
- we have not even talked about package.json, will this also be a problem?

Reply:
- `index.ts` should actually be named `bot.ts` - not a problem.
- `bot.ts` should actually have different content - not a problem again. Of course user should write default code for bot launching
- `tsconfig.json` is wrong - what's wrong with it?
- we have not even talked about package.json, will this also be a problem? - Not actually. package.json is:

```JSON
{
  "name": "test",
  "module": "index.ts",
  "type": "module",
  "dependencies": {
    "grammy": "^1.30.0"
  },
  "devDependencies": {
    "@types/bun": "latest"
  },
  "peerDependencies": {
    "typescript": "^5.0.0"
  }
}
```

Reply:
![image](https://github.com/user-attachments/assets/cd9664c8-8a1d-4f2a-80a7-557614cc48b0)

Reply:
Why is TS a peer dep? It should be a dev dep.

Reply:
Bun adds a tiзуscript to Peer Deps, but honestly I haven't tried to figure out the behaviour of that. We could move it to dev deps and nothing would change.